sandbox/src/services/patient-searcher.js

```diff
@@ -64,11 +64,17 @@ function buildPatientResponse(examplePatients = [], searchScore = 1.0) {
                 resource: patient,
             })
         });
+        return response;
     } else {
-        delete response.entry
-    }
+        return {
+            resourceType: "OperationOutcome",
+            timestamp: datefns.format(Date.now(), "yyyy-MM-dd'T'HH:mm:ss+00:00"),
+            "issue": [{"severity": "warning", "code": "not-supported", 
+            "details": {"coding": [{"system": "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode", 
+            "version": "1", "code": "not-supported", "display": "Unsupported operation"}]}, 
+            "diagnostics": "This mock endpoint has no example response for this combination of search parameters"}]}
+        }
```

Comment:
This final `}` is very misleading as it's for the else but lined up with the return. Whatever happens with the rest of the formatting I think this should be one indentation less